nrcatalogtools/lvc.py

```diff
@@ -107,21 +102,21 @@ def get_modes_from_lvcnr_file(path_to_file,
 
         if h5file.attrs["Format"] < 3:
             spin_args = lalsim.SimInspiralNRWaveformGetSpinsFromHDF5File(
-                -1, Mtot, path_to_file)
+                -1, Mtot, path_to_file
+            )
         else:
             spin_args = lalsim.SimInspiralNRWaveformGetSpinsFromHDF5File(
-                f_low, Mtot, path_to_file)
+                f_low, Mtot, path_to_file
+            )
 
-        mass_args = list(
-            mtotal_eta_to_mass1_mass2(Mtot * MSUN, h5file.attrs["eta"]))
+        mass_args = list(mtotal_eta_to_mass1_mass2(Mtot * MSUN, h5file.attrs["eta"]))
```

Comment:
where is this `MSUN` coming from? It should be `lal.MSUN_SI`.

Reply:
... just checked the whole file - there is no definition for `MSUN`.

Reply:
`flake8` had flagged these issues as well and I have corrected them in #13.